Cisco/iOS-XR/snmp.conf

```diff
@@ -1,2 +1,10 @@
-! making sure ifIndex gets persisted with SNMP after router reboot 
+! SNMP IOS-XR
+!
+! Set SNMP Access ACL
+ipv4 access-list 33
+32 remark NNS SNMP Access
```

Comment:
I believe it should rather be:
`33 remark NNS SNMP Access `
given that the ACL below says 33 ?

Also same comment that for iOS-XE config RE: remark statement:
`access-list 33 remark Kentik SNMP Polling ACL US and EU`

Reply:
Indeed!